share/rocm/cmake/ROCMCreatePackage.cmake

```diff
@@ -160,6 +160,15 @@ macro(rocm_create_package)
     if(ROCM_VERSION_NUM)
         set(CPACK_PACKAGE_VERSION "${CPACK_PACKAGE_VERSION}.${ROCM_VERSION_NUM}")
     endif()
+    # Set the License to rpm header
+    if(PARSE_LICENSE)
+        set( CPACK_RPM_PACKAGE_LICENSE "${PARSE_LICENSE}" )
+    endif()
+    # Setting the CPACK_RESOURCE_FILE_LICENSE variable.
+    # Installing license file to /opt/rocm/share/doc/<component_name> will happen in rocm_setup_license macro
```

Comment:
Where is this `rocm_setup_license` at? I dont see any documentation on it.

Reply:
It is located on line 326, it does still need to be documented. Apologies on that, this feature was requested to be enabled ASAP and documentation was overlooked.

Reply:
Ok, looking closer this is called only when `COMPONENTS` are passed. Also, if users are supposed to call it, then the interface should be changed to use named paramaters instead of passing a bool.

Reply:
Opened #67 to resolve issues with `rocm_setup_license`, as they are unrelated to this PR.